apps/server/src/modules/files-storage/mapper/ifile-builder.builder.ts

```diff
@@ -0,0 +1,42 @@
+import { AxiosResponse } from 'axios';
+import { FileInfo } from 'busboy';
+import { Request } from 'express';
+import internal, { Readable } from 'stream';
+import { IFile } from '../interface';
+
+export class IFileBuilder {
+	private static buildFile(
+		name: string,
+		buffer: Buffer | Blob | ReadableStream | Readable,
+		size: number,
+		mimeType: string
+	): IFile {
+		const file: IFile = {
+			name,
+			buffer,
+			size,
+			mimeType,
+		};
+
+		return file;
+	}
+
+	public static buildFromRequest(
```

Comment:
IFileBuilder.buildFromRequest double build in name. Let look in next ticket.

Reply:
Taken to BC-2553